docs/src/main/asciidoc/reactive-sql-clients.adoc

```diff
@@ -88,8 +88,16 @@ Otherwise, you can manually add this to the dependencies section of your `pom.xm
 </dependency>
 ----
 
+=== Mutiny
+
+Reactive REST endpoints in your application that return Uni or Multi need `Mutiny support for RESTEasy` extension (`io.quarkus:quarkus-resteasy-mutiny`) to work properly:
+
+[source,shell]
+----
+./mvnw quarkus:add-extension -Dextensions="resteasy-mutiny"
+----
+
 [TIP]
-.Mutiny
 ====
```

Comment:
Thanks for the PR @tivrfoa.

I am wondering if this should be part of the `TIP`?

/cc @cescoffier @gsmet

Reply:
Thanks @machi1990 

Both versions below so you can decide what looks better:

https://github.com/tivrfoa/quarkus/blob/mention-need-of-resteasy-mutiny-extension-in-reactive-sql-clients-adoc/docs/src/main/asciidoc/reactive-sql-clients.adoc

https://github.com/tivrfoa/quarkus/blob/in-tip/docs/src/main/asciidoc/reactive-sql-clients.adoc

Reply:
I would keep it like that, looks good to me!

Reply:
Fair enough.